source/sass/components/primary-nav.scss

```diff
@@ -35,6 +35,7 @@
     line-height: calc(23 / 17);
```

Comment:
Hey Simon! it looks like line 225 of this file is pushing down the PNI nav bar _just_ a bit, to where you can see the search bar if its opened:


![Screenshot 2022-08-29 at 19-42-32 Privacy Not Included A Buyer’s Guide for Connected Products](https://user-images.githubusercontent.com/18314510/187337032-120795f2-516a-4530-95cf-77fd3781d950.png)